.docker/Dockerfile

```diff
@@ -59,5 +64,5 @@ COPY --from=build ["/library_pkgs", "/library_pkgs"]
 
 RUN apt-get update && \
     apt install -y /library_pkgs/aravis_0.6.4-1_amd64.deb && \
-    apt install /library_pkgs/mescan_0.2.4-1_amd64.deb && \
-    apt install /library_pkgs/llt_0.2.4-1_amd64.deb
\ No newline at end of file
+    apt install /library_pkgs/mescan_${SCANCONTROL_SDK_VERSION}-1_amd64.deb && \
+    apt install /library_pkgs/llt_${SCANCONTROL_SDK_VERSION}-1_amd64.deb
```

Comment:
```suggestion
    apt install /library_pkgs/llt_${SCANCONTROL_SDK_VERSION}-1_amd64.deb

```

Reply:
Add a white line at the end of the file? I do not see any changes otherwise ;)

Reply:
Yes. You're file is missing an empty last line.

Github even tells you so ;)

Reply:
If that is true, I am just curious about the why behind this

Reply:
Ah, I see the warning now indeed. Did not know this was a thing

Reply:
https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file